bigbluebutton-html5/imports/api/users/server/methods/userLeaving.js

```diff
@@ -30,11 +30,11 @@ export default function userLeaving(credentials, userId) {
       'user-not-found', `You need a valid user to be able to toggle audio`);
   }
 
-  if (User.user.connection_status === OFFLINE_CONNECTION_STATUS) {
+  if (User.connection_status === OFFLINE_CONNECTION_STATUS) {
```

Comment:
Its undefined because you are using arrow functions and losing the context, try to switch from `() => {` to `function() {` on line 7.

Could you print it as a JSON instead of the logger parser? Kinda strange that is not nested since the `setConnectionStatus` modifier update the key `'user.connection_status'` :thinking: